src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/ModellingGroupRouteConfig.kt

```diff
@@ -22,6 +22,9 @@ object ModellingGroupRouteConfig : RouteConfig
                     .json()
                     .post()
                     .secure(writePermissions),
```

Comment:
Is this a reasonable route? I can't really use /modelling-groups/memberships as it will tend to confuse the routing (it would interpret 'memberships' as a :group-id for the other route.